studies.py

```diff
@@ -316,3 +316,50 @@ def objective(trial: optuna.Trial) -> float:
     )
     study.optimize(objective, n_trials=50, timeout=600)
     return study
+
+
+def create_single_objective_studies_for_timeline() -> List[Tuple[str, StudiesType]]:
+    studies: List[Tuple[str, StudiesType]] = []
+    storage = optuna.storages.InMemoryStorage()
+
+    def objective_timeline(trial: optuna.Trial) -> float:
+        x = trial.suggest_float("x", 0, 1)
+        time.sleep(x * 0.1)
+        if x > 0.8:
+            raise ValueError()
+        if x > 0.4:
+            raise optuna.TrialPruned()
+        return x**2
+
+    # Single-objective study
+    study = optuna.create_study(
+        study_name="A single objective study consuming time",
+        storage=storage,
+    )
+
+    study.enqueue_trial({"x": 0.3})
+    study.enqueue_trial({"x": 0.9})
+    study.enqueue_trial({"x": 0.5})
```

Comment:
[nit] Could you add code comments to explain the expected trial states.
```suggestion
    study.enqueue_trial({"x": 0.3})  # Add a COMPLETE trial.
    study.enqueue_trial({"x": 0.9})  # Add a FAIL trial.
    study.enqueue_trial({"x": 0.5})  # Add a PRUNED trial.
```